src/App.js

```diff
@@ -0,0 +1,32 @@
+import Stack from "@mui/material/Stack";
+import Button from "@mui/material/Button";
+import logo from "./logo.svg";
+import "./App.css";
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+        <Stack spacing={2} direction="row">
```

Comment:
This is where I imported the button component from material for testing purposes.

Reply:
Great! Good work